r/tests/testthat/test-feather.R

```diff
@@ -196,3 +196,19 @@ test_that("Character vectors > 2GB can write to feather", {
 })
 
 unlink(feather_file)
+
+ft_file <- system.file("v0.7.1.feather", package = "arrow")
+
+test_that("Error messages are shown when the compression algorithm lz4
+          is not found", {
+  skip_on_cran()
```

Comment:
Why are these tests `skip_on_cran()`?

Reply:
I just wanted to keep it fast, I've removed this a few days ago